package.json

```diff
@@ -51,11 +51,11 @@
     "commitmsg": "npm run --silent lint:commitmsg",
     "bootstrap": "echo 'npm run bootstrap is no longer required. Hooray!'",
     "clean": "lerna clean",
-    "build": "npm run --silent grunt:concurrent -- build",
+    "build": "./tooling/build.js",
     "check-karma-output": "./scripts/analyze-output.sh",
     "check-package": "if [ -z ${PACKAGE+x} ]; then echo 'environment variable PACKAGE must be defined' && exit 1; fi",
     "prebuild:package": "npm run --silent check-package",
-    "build:package": "npm run --silent grunt:package -- build",
+    "build:package": "echo 'deprecated; just use \"npm run build\"'./tooling/build.js",
```

Comment:
Is the `build.js` portion needed? Doesn't `npm run build` do that automatically?